freqtrade/configuration/timerange.py

```diff
@@ -42,9 +42,8 @@ def parse_timerange(text: Optional[str]):
                   (r'^-(\d{10})$', (None, 'date')),
                   (r'^(\d{10})-$', ('date', None)),
                   (r'^(\d{10})-(\d{10})$', ('date', 'date')),
-                  (r'^(-\d+)$', (None, 'line')),
-                  (r'^(\d+)-$', ('line', None)),
-                  (r'^(\d+)-(\d+)$', ('index', 'index'))]
+                  (r'^(\d{13})-(\d{13})$', ('date', 'date')),
```

Comment:
Then it should also recognize the formats for 
`1231006505000-` and `-1233360000000`, besides `1231006505000-1233360000000`

Reply:
arrrgh yes :D